src/Compilers/Core/Portable/Emit/NoPia/CommonEmbeddedMethod.cs

```diff
@@ -240,6 +240,8 @@ Cci.IDefinition Cci.IReference.AsDefinition(EmitContext context)
                 return this;
             }
 
+            string Cci.IMethodDefinition.Name => Name;
```

Comment:
What's this?

Reply:
I moved `IGlobalMethodDefinition.Name` to `IMethodDefinition.Name`.

Reply:
I'm reverting this change.